terraform/modules/jaspersoft/setup_script.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+export TOMCAT_VERSION=9.0.65
```

Comment:
Jaspersoft doesn't seem to be compatible with the latest 9.x version (9.0.67). It's only just released, so hopefully it's just a Tomcat bug that will get fixed.